src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -84,6 +91,8 @@ public function setInputStream($stream)
     /**
      * Returns the helper's input stream.
      *
+     * @deprecated Will be removed in 4.0, use StreamableInputInterface::getStream() instead
```

Comment:
That does not work like that. You MUST add the deprecation notice and fix the tests accordingly. But instead of doing that, I would recommend to just keep `getInputStream()` and add `setInputStream()`.